### ISSUE TITLE
Captcha after every request

### ISSUE THREAD

**Issue by NarekBektashyan** [kovinevmv/getcontact#48]
Maybe after some update of getcontact  the script doesn't work correctly, it returns no tags, only one name and then captcha . Any idea how to fix that?

**Comment by kovinevmv** [comment]
Yes, they already wrote to me that now often captcha appears, which needs to be bypassed, I plan to fix it, but I do not know when I will find the time

**Comment by NarekBektashyan** [comment]
And maybe I can help you? I think the app need a new reverse engineering. If youre interested in my help please write me narekbek1@gmail.com or in telegram @Naarek

**Comment by y-cherdantsev** [comment]
Мало того, у меня возникла ситуация, когда после нескольких неверно решенных запросов апишка гетконтакта перестает воспринимать даже верные решения. Когда такое случается можно смело забыть на сутки об используемом в тот момент токене пока ситуация не нормализуется. Не совсем понятно правильно ли я увидел закономерность, но если да то не понятно, почему сами разработчики приложения не фиксят эту проблему. Либо это у меня на стороне клиента какой-либо косяк. Если есть какие-то мысли по этому поводу был бы рад выслушать. (Не хотел выносить в отдельное обсуждение, потому-что по ощущениям проблема появилась как-раз после изменений которые обсудили выше)

**Comment by kovinevmv** [comment]
Это проблема не на моей стороне. Я уже писал об этом
https://github.com/kovinevmv/getcontact#update-01022021

**Comment by wahyusaputra** [comment]
I used my own token as suggested. It was giving me results for 3 attempts (3 requests for 3 phone numbers). However, afterwards it gives me:
"Captcha is detected" and "Try to decode" messages on the log.

Would you please kindly give more information regarding the token required for your script to work?

Thank you.

**Comment by kovinevmv** [comment]
The problem arises if you send a lot of requests, and only for number identification. A captcha pops up and you need to solve it. However, sometimes when you enter the text correctly, but it appears that it did not pass. Even if you go through the mobile app, you will see form for enter captcha and again the same situation, cant pass it. I agree that you just need to forget about the account for a while (1 day). To see captcha less often, you need to send additional requests - get a user, and so on.

**Comment by umsterdam** [comment]
How To Edit Code ?
import argparse

from getcontact.getcontact import GetContactAPI
from getcontact.config import config

description = "Get information about phone number from databases of GetContact"

parser = argparse.ArgumentParser(description=description,
                                 formatter_class=argparse.RawTextHelpFormatter)
parser.add_argument("-p",
                    "--phone",
                    help="Phone number (example: +78005553535)")
parser.add_argument("-j",
                    "--json",
                    help="Print output in JSON format",
                    action="store_true")
parser.add_argument("-v", "--verbose", help="Log", action="store_true")

args = parser.parse_args()

phone = args.phone
getcontact = GetContactAPI()
config.VERBOSE = args.verbose

if args.json:
    print(getcontact.get_information_by_phone(phone))
else:
    getcontact.print_information_by_phone(phone)
#######################################################
Phone: None
User: Not Found

**Comment by yagamicell** [comment]
create X- req signature not working.. because for now i see in request header use X-Req-Timestamp: with miliseconds time 13 digits unixtime not 10 digits

**Comment by aunicorn111** [comment]
Не уверена, что проблема вечной капчи на стороне разработчика. В моём случае код впадает в вечную капчу и не выходит из неё даже если попытаться решить её из эмулятора. К сожалению это также и не проходит спустя день и даже два... Как это лечить неясно. Возможно только сменой эмулятора или что-то типа того??

**Comment by oifj34f34f** [comment]
Is GetContact paid now?


<details>
  <summary>Screenshot</summary>

  <img src="https://github.com/user-attachments/assets/7dc9023a-c599-407f-92a4-6cd0049689e6" width="400">

</details>

**Comment by kovinevmv** [comment]
> Is GetContact paid now?
> 
> Screenshot

Hello, no, its still free, only extra functions are paid
